apps/questions/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db.models import (
+    CharField,
+    CASCADE,
+    ForeignKey,
+    ManyToManyField,
+    DateTimeField,
+    EmailField,
+)
+
+from apps.lib.models import UUIDModel
+
+# Create your models here.
+class Question(UUIDModel):
+    question_text = CharField(max_length=200)
```

Comment:
the name of this should be just `text`, because it's in the Question model. Therefore, its assumed to be the text of the question.